contrib/parseq-restli-client/src/main/java/com/linkedin/restli/client/GetRequestGroup.java

```diff
@@ -316,7 +320,8 @@ public void onSuccess(Response<RT> responseToGet) {
   }
 
   @Override
-  public <RT extends RecordTemplate> void executeBatch(final RestClient restClient, final Batch<RestRequestBatchKey, Response<Object>> batch) {
+  public <RT extends RecordTemplate> void executeBatch(final RestClient restClient, final Batch<RestRequestBatchKey, Response<Object>> batch,
```

Comment:
GetRequestGroup is not a public class and is considered implementation detail.
The usage you pointed is just a copy of exactly the same class on GitHub :-)